### ISSUE TITLE
The parameters in "Quick Start" are not implemented yet (grayed out)

### ISSUE THREAD

**Issue by afbjorklund** [afbjorklund/lima-gui#6]
Use the "Edit YAML" as a workaround, for now. See "default.yaml" for options

https://github.com/lima-vm/lima/blob/master/pkg/limayaml/default.yaml

```yaml
# Arch: "default", "x86_64", "aarch64".
# Default: "default" (corresponds to the host architecture)
arch: "default"

# CPUs: if you see performance issues, try limiting cpus to 1.
# Default: 4
cpus: 4

# Memory size
# Default: "4GiB"
memory: "4GiB"

# Disk size
# Default: "100GiB"
disk: "100GiB"
```

**Comment by afbjorklund** [comment]
It is possible to get the default values, from `limactl info`.

```json
        "cpus": 4,
        "memory": "4GiB",
        "disk": "100GiB",
```

**Comment by afbjorklund** [comment]
The vmTypes are also available, since lima version 0.14.2

```yaml
    "vmTypes": [
        "qemu"
    ]
```

The cpuTypes are not exported yet, it's a text field for now.

**Comment by afbjorklund** [comment]
The available architectures are currently hardcoded.

```go
type Arch = string

const (
        X8664   Arch = "x86_64"
        AARCH64 Arch = "aarch64"
        RISCV64 Arch = "riscv64"
)
```

The emulator-only RISC-V was added in version 0.11.0

**Comment by afbjorklund** [comment]
Fixed, 62d31a99b0251135366f8125fc03fa2419c5284c
